nilearn/plotting/tests/test_matrix_plotting.py

```diff
@@ -21,6 +21,14 @@ def test_matrix_plotting():
     # test if it returns an AxesImage
     ax.axes.set_title('Title')
     plt.close()
+    # test if an empty list works as an argument for labels
+    ax = plot_matrix(mat, labels=[])
```

Comment:
could we also test with `labels=None` (the default)?

Reply:
Sure, added the test.

Reply:
thanks!